docs/blog/posts/how_to_uncompress_large_files.md

```diff
@@ -0,0 +1,71 @@
+---
+date:
+  created: 2023-08-18
+categories:
+    - zip
+authors:
+    - nrapstin
+---
+
+
+Compressed files are widely used to save space and facilitate file transfers. Understanding how to uncompress these files ensures quick access to your data, no matter the format. This post will guide you through the process of uncompressing files on the Yens.
+
+
+<!-- more -->
+
+# How do I extract compressed files?
```

Comment:
This can be the title but let's move it all the way up to the top of the page (and capitalize the words)